src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,27 @@
+package duke;
+
+import java.util.Scanner;
+
+//This class has a chatbot which will generate a list based on a set of tasks
+//that have been input by the user. These tasks include deadlines, events and ToDos
+//The user input will be parsed to extract the date and time of the events and deadlines
```

Comment:
Perhaps a multiline comment might be better, e.g.
```suggestion
/**
 * This class has a chatbot which will generate a list based on a set of tasks
 * that have been input by the user. These tasks include deadlines, events and ToDos
 * The user input will be parsed to extract the date and time of the events and deadlines
 */
```